makefile

```diff
@@ -17,6 +17,13 @@ include sources/build/boards/$(ARCH)-$(BOARD)/build.mk
 
 export LC_ALL=C
 
+ifeq '$(TOOLCHAIN)' 'gnu'
```

Comment:
I just turned them into warning to be able to compile the code.
I don't see why returning a 64 bit integer causes the oob right now.
The error is not introduced by the changes of the commit.
It might be possible that the new gcc version and the compiler flags are causing it.